wordpress-mobile/WordPress-Editor-iOS
# Migrates the editor to WKWebView.

Closes #260.

I'm marking this as "not ready" until we can do some proper testing.

Need some help testing this: @rachelmcr, @bummytime, @SergioEstevao.

**How to test:**
- WPiOS branch is `editor/260-wkwebview`.
- Run this editor alongside the old one and make sure there are no regressions.
- Bugs found in both the old editor and the new one should be ignored.
